table/src/main/java/com/bazaarvoice/emodb/table/db/curator/TableMutexManager.java

```diff
@@ -13,23 +13,18 @@
     private static final int NUM_MUTEXES = 256;
 
     private final Mutex[] _mutexes;
-    private final CuratorMutex _oldMutex;
 
-    public TableMutexManager(CuratorFramework curator, String oldPath, String newPath) {
+    public TableMutexManager(CuratorFramework curator, String path) {
         _mutexes = new Mutex[NUM_MUTEXES];
         for (int i = 0; i < NUM_MUTEXES; i++) {
-            _mutexes[i] = new CuratorMutex(curator, ZKPaths.makePath(newPath, Integer.toString(i)));
+            _mutexes[i] = new CuratorMutex(curator, ZKPaths.makePath(path, Integer.toString(i)));
         }
-        _oldMutex = new CuratorMutex(curator, oldPath);
     }
 
     public void runWithLockForTable(Runnable runnable, Duration acquireTimeout, String table) {
 
-
-        _oldMutex.runWithLock(() -> {
-            _mutexes[Math.abs(Hashing.murmur3_128().hashString(table, Charsets.UTF_8).asInt() % NUM_MUTEXES)]
-                            .runWithLock(runnable, acquireTimeout);
-        }, acquireTimeout);
+        _mutexes[Math.abs(Hashing.murmur3_128().hashString(table, Charsets.UTF_8).asInt() % NUM_MUTEXES)]
```

Comment:
This isn't a comment on this PR but it's something that probably should have been part of the first PR, and now seems like as good of place as any to address it.

There's no null check on `table`, and hashing a null String will throw an NPE.  I don't know if it's even possible in the code path but probably best to defensively check of null and assign null a consistent partition.  Could be something as simple as hashing the result of `Strings.nullToEmpty(table)` or  something like `int which = table == null ? 0 : Math.abs(...)`